src/docker-in-docker/README.md

```diff
@@ -17,6 +17,7 @@ Create child containers *inside* a container, independent from the host's docker
 |-----|-----|-----|-----|
 | version | Select or enter a Docker/Moby Engine version. (Availability can vary by OS version.) | string | latest |
 | moby | Install OSS Moby build instead of Docker CE | boolean | true |
+| mobyBuildxVersion | Select or enter a Moby Buildx version. (Availability can vary by OS version.) | string | latest |
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!